### PR TITLE
Feedback menu

### PR DESCRIPTION
**Pull request by mtthwryn**
refactored menu to be a separate function, added new menu item "Feedback", takes user feedback and sends it to the admin group

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -49,37 +75,18 @@ async def start_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
...
 
     await update.message.reply_text(welcome_message, reply_markup=reply_markup)
 
 async def help_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
-    await update.message.reply_text("What do you want")
-
-# Responses
-# AI will go in here later on to process inputs
-
-def handle_response(text: str) -> str:
```
Comment (TayKaiJun) [inline src/main.py:67]: looks like you refactored this into the handle_message function? can you refactor it back out cos we'll eventually implement some kind of response handler (either chat AI or some fixed set of responses) but yea, it'll be better for future development

File: src/main.py [hunk 2]
```diff
@@ -95,11 +102,23 @@ async def handle_message(update: Update, context: ContextTypes.DEFAULT_TYPE):
...
+            response = "Thank you for your feedback! Ah Gong has sent your input to my grandchildren working on improving my services."
+            # Reset the state to None after handling the feedback
+            context.user_data["state"] = None
+        else:
+            response: str = handle_response(text)
         
     print("Bot: ", response)
     await update.message.reply_text(response)
 
+# Callback query handler
```
Comment (TayKaiJun) [inline src/main.py:119]: `Callback query` refers to the menu buttons? maybe renaming this as `Menu Buttons` might be clearer unless there are stuff that's also categorized under callback query?

**Review by TayKaiJun — CHANGES_REQUESTED** [review]
minor refactoring suggestions to upkeep good developmental practices

**Comment by TayKaiJun** [comment]
Just to pen it down, we'll probably need to start looking at abstracting some of the handlers in the main.py as the bot gets more developed.

File: src/main.py (hunk 2 above)
Comment (mtthwryn) [reply src/main.py:119]: Will rename to `Menu Buttons`

File: src/main.py (hunk 1 above)
Comment (mtthwryn) [reply src/main.py:67]: it is currently still outside the handle_message function, idk why github shows the additions and deletions in a weird way but this is currently what it is for the help command

```
async def help_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
    # Call the function to create the menu
    reply_markup = create_menu()

    await update.message.reply_text("Here are our available menus:", reply_markup=reply_markup)
```

**Pull request merged**